### PR TITLE
Methods for DidlObject.resources[0].uri

### PR DESCRIPTION
**Pull request by ghcs27**
I have noticed that I use DidlObject.resources[0].uri quite often and it is also present in the SoCo code multiple times. Hardcoding the index [0] in so many places does not seem very good practice to me.

Therefore, I have added get_uri and set_uri methods to DidlObject as a shortcut.

### CODE REVIEW

File: soco/data_structures.py [hunk 1]
```diff
@@ -690,6 +690,40 @@ def to_element(self, include_namespaces=False):
 
         return elt
 
+    def get_uri(self, resource_nr=0):
+        """The uri to use for playing this item.
+
+        Args:
+            resource_nr (int): The index of the resource. Note that there is no
+            known object with more than one resource, so you can probably keep
```
Comment (KennethNielsen) [inline soco/data_structures.py]: When writing these docstring in the Napoleon style for Sphinx, a continued section should be indented once further. So it should be:
```python
    """
    Args:
        resource_nr (int): Very long multiline description of what it does
            it even continues on this line
            and this one as well.
    """
```

File: soco/data_structures.py [hunk 2]
```diff
@@ -690,6 +690,40 @@ def to_element(self, include_namespaces=False):
...
+            the default value (0).
+        """
```
Comment (KennethNielsen) [inline soco/data_structures.py]: Remember the return value in the docstring.

File: soco/data_structures.py [hunk 3]
```diff
@@ -690,6 +690,40 @@ def to_element(self, include_namespaces=False):
 
         return elt
 
+    def get_uri(self, resource_nr=0):
+        """The uri to use for playing this item.
```
Comment (KennethNielsen) [inline soco/data_structures.py]: According to pep 257, the first single line description should preferably be kept on the form of describing the methods effect e.g. """Return the uri .."""

File: soco/data_structures.py [hunk 4]
```diff
@@ -690,6 +690,40 @@ def to_element(self, include_namespaces=False):
...
+
+        Args:
+            resource_nr (int): The index of the resource. Note that there is no
+            known object with more than one resource, so you can probably keep
+            the default value (0).
+        """
+        return self.resources[resource_nr].uri
+
+    def set_uri(self, uri, resource_nr=0, protocol_info=None):
```
Comment (KennethNielsen) [inline soco/data_structures.py]: All good.

**Comment by ghcs27** [comment]
Docstring updated!

**Comment by KennethNielsen** [comment]
Looks good. I'll merge.

BTW @ghcs27 if you feel like it, I have a couple a small documentation PR's waiting for review ;)

**Comment by KennethNielsen** [comment]
@ghcs27 can you add a comment about this change to the release notes thread?

**Pull request merged**